components/script/Cargo.toml

```diff
@@ -75,6 +75,7 @@ mime = { workspace = true }
 mime_guess = { workspace = true }
 msg = { workspace = true }
 net_traits = { workspace = true }
+num_cpus = "1.1.0"
```

Comment:
I'm not sure this needs a bug as in the Servo project we don't necessarily create a bug for every feature or issue.

Now that `num_cpus` is used in `config` and `script` can you make it a workspace dependency so that it's easier to keep it up to date? Please don't modify `style` though as that is a vendored version of Mozilla's style and will probably soon move out of the source tree.

Reply:
I converted it to a workspace dependency.